depthai_demo.py

```diff
@@ -259,6 +260,11 @@ def create_nn_pipeline(self, p, nodes):
             xin.setStreamName(self.input_name)
             xin.out.link(nn.input)
             setattr(nodes, self.input_name, xout)
+            # Send the video frame back to the host
+            if conf.args.sync:
+                nodes.xout_rgb = p.createXLinkOut()
+                nodes.xout_rgb.setStreamName("host")
```

Comment:
Let's add a `host` to Previews - https://github.com/luxonis/depthai/blob/1f2244c75a660272bdef15f1865a4fda8b3fcd6a/depthai_demo.py#L59 and use `Previews.host.name` as a stream name